app/autocomplete_mention/routes.test.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var expect = require('chai').expect
+  , request = require('supertest');
+
+var app = require('../../app');
+
+describe('Autocomplete', function() {
+  describe('GET /mention', function() {
+    it('should send a response', function(done) {
+      request(app)
+        .get('/autocomplete/mention')
+        .expect(200)
+        .end(function(err,res) {
+          if(err) return done(err);
+          expect(res).to.exist;
+          done();
+        });
+    });
+
+    it('should respond with json', function(done) {
+      request(app)
+        .get('/autocomplete/mention?pattern=wor')
+        .set('Accept','application/json')
+        .expect('Content-Type',/json/)
+        .expect(200,done);
```

Comment:
@sscotth I changed the test above the .json test, so I felt this test was still relevant - do you agree, or should I delete this test?